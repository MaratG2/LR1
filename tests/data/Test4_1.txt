-330923728899499857069191781959446083154.003452
-314437307656796349630567841180932441663.426378
-314330483089388410159109737465066742423.331873
-304684001272342249356367351620614975260.037730
-287803552195116492398701908314061551709.106142
-280803682330637483869873011144831498476.502512
-277273062254894935627258198038341522688.251842
-269122839159530677815763040971091462928.946311
-267730195590775848015352528277629505107.801036
-262208542098683159348711264813974445662.949846
-259072407956627105894845164866999830064.399736
-255193838490414259328966000164232576024.928088
-252297336577484272862835720185706575497.982892
-240356106236449449716723505141021943160.546797
-187673145314598749987780492104407357508.489854
-178973351398121736429607457784008556256.794349
-175333978621555934146591470933825621404.934677
-159688381615327027054957802001914373564.821308
-140850394658377680116652148571066105155.794636
-95621627853223928675829681228874928959.017009
-91456152908968463379289708960656556791.080439
-66817561119249827513144909492290291340.451400
-51232158705229669945432584339272231065.705970
-49086906455879309534694020486211737161.306959
-34871696196541663822339201543700863753.001044
-6574487021088714007340606744885482862.415383
14089134377506070290304050662859675234.447947
55288403390862540148090452705122077369.198609
73629631723391814236510519933849658126.962184
79417098700652381807202109346224029130.237712
86690921525066450122322089894526718491.721175
99342250996041227513294758505819247893.828411
124145359437878691868208716560932558411.603534
126183011951957243187063289175172573649.437672
161560624960662200659572724123471010702.105002
172533942846867227827655600303557603911.669938
230988437026302372150289635440085889221.979652
232935762334770758903337242759625852211.572604
240941436948948544354139753775766570593.642460
273910129733653675587223611402021596620.848112
278993079762184792093481922342951592913.861906
307635720979012922971216278159379342560.183609
325576116239496630397429401886731357767.978281