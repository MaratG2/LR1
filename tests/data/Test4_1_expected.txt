-3.3092374E+38
-3.144373E+38
-3.1433048E+38
-3.04684E+38
-2.8780356E+38
-2.8080367E+38
-2.7727305E+38
-2.6912283E+38
-2.677302E+38
-2.6220854E+38
-2.5907241E+38
-2.5519385E+38
-2.5229733E+38
-2.403561E+38
-1.8767314E+38
-1.7897334E+38
-1.7533399E+38
-1.5968838E+38
-1.4085039E+38
-9.562162E+37
-9.145615E+37
-6.681756E+37
-5.123216E+37
-4.9086905E+37
-3.4871696E+37
-6.574487E+36
1.4089134E+37
5.5288403E+37
7.362963E+37
7.94171E+37
8.669092E+37
9.934225E+37
1.2414536E+38
1.2618302E+38
1.6156063E+38
1.7253394E+38
2.3098843E+38
2.3293577E+38
2.4094144E+38
2.7391013E+38
2.7899308E+38
